src/olympia/editors/models.py

```diff
@@ -78,7 +78,7 @@ def new_editors():
 
         return [dict(user=i.arguments[1],
                      created=i.created)
-                for i in items]
+                for i in items if i.arguments[1] in group.users.all()]
```

Comment:
> Ideally, I'd like to do something like .filter(_arguments[1]__in=group.users.all()) but of course that doesn't work. Is there a way to do that?

Sadly, no. Postgres > 9 and MySQL > 5.7 support JSON fields and querying them in various ways, but Django only supports the Postgres variant (plus we'd need Django 1.9, and a migration... yeah, not going to happen any time soon)